doc/src/manual/arrays.md

```diff
@@ -747,15 +750,34 @@ sparse data structure, including structural nonzeros. In order to count the exac
 values that are nonzero, use [`countnz()`](@ref), which inspects every stored element of a sparse
 matrix.
 
-### Sparse matrix constructors
+### Sparse Vector Storage
 
-The simplest way to create sparse matrices is to use functions equivalent to the [`zeros()`](@ref)
-and [`eye()`](@ref) functions that Julia provides for working with dense matrices. To produce
-sparse matrices instead, you can use the same names with an `sp` prefix:
+Sparse vectors are simply constructed by two vectors; one for the stored values and one for
```

Comment:
Perhaps instead "Sparse vectors are stored in a close analog to compressed sparse column format for sparse matrices."? (The preceding seems sufficient and informative given the presence of the `SparseVector` `struct` just below.)